android/app/src/main/java/com/hotwirerndemo/RNSession.kt

```diff
@@ -5,6 +5,58 @@ import android.widget.FrameLayout
 import androidx.appcompat.app.AppCompatActivity
 import com.facebook.react.bridge.ReactContext
 import dev.hotwire.turbo.session.TurboSession
+import dev.hotwire.turbo.views.TurboView
 import dev.hotwire.turbo.views.TurboWebView
+import dev.hotwire.turbo.visit.TurboVisit
 
-class RNSession(context: Context) : FrameLayout(context)
\ No newline at end of file
+class RNSession(context: Context) : FrameLayout(context) {
+
+    lateinit var session: TurboSession private set
+    private val reactContext = context as ReactContext
+    private val registeredVisitableViews = mutableListOf<SessionSubscriber>()
+
+    init {
+        setupNewSession()
+    }
+
+    private fun setupNewSession() {
+        val activity = reactContext.currentActivity as AppCompatActivity
+        val webView = TurboWebView(context, null)
+
+        session = TurboSession("testSessionName", activity, webView)
```

Comment:
Should the TurboSession name also be dynamic and based on the RNSession name?

Reply:
> The name of the TurboSession (this is arbitrary, but must be unique in your app)
^ From docs ^
I will make it unique then.